<!DOCTYPE html>
<!-- saved from url=(0051)https://plants.ensembl.org/info/data/ftp/index.html -->
<html lang="en-gb" class=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script async="" src="./ENSEMBL_FTP_Download_files/js"></script>
  <title>FTP Download</title>

<!--[if lte IE 7]><link rel="stylesheet" type="text/css" media="all" href="/minified/f38b8df6fcf96e48eb2d592efce9d511.ie7.css"/><![endif]--><!--[if gt IE 7]><link rel="stylesheet" type="text/css" media="all" href="/minified/f38b8df6fcf96e48eb2d592efce9d511.css"/><![endif]--><!--[if !IE]><!--><link rel="stylesheet" type="text/css" media="all" href="./ENSEMBL_FTP_Download_files/f38b8df6fcf96e48eb2d592efce9d511.css"><!--<![endif]--><!--[if lte IE 7]><link rel="stylesheet" type="text/css" media="all" href="/minified/615221592b22312413d87118e6775a7e.image.ie7.css"/><![endif]--><!--[if gt IE 7]><link rel="stylesheet" type="text/css" media="all" href="/minified/615221592b22312413d87118e6775a7e.image.css"/><![endif]--><!--[if !IE]><!--><link rel="stylesheet" type="text/css" media="all" href="./ENSEMBL_FTP_Download_files/615221592b22312413d87118e6775a7e.image.css"><!--<![endif]-->
  <link href="https://plants.ensembl.org/i/ensembl-favicon.png" type="image/png" rel="icon">
  <link type="image/png" rel="apple-touch-icon" href="https://plants.ensembl.org/apple-touch-icon.png">
  <link rel="search" type="application/opensearchdescription+xml" title="Ensembl (All)" href="https://plants.ensembl.org/opensearch/all.xml">

  <meta name="description" content="Ensembl Plants is a genome-centric portal for plant species of scientific interest">
<meta name="viewport" content="target-densitydpi=device-dpi, width=device-width, initial-scale=1.0, maximum-scale=2.0, user-scalable=yes">
<script async="" src="./ENSEMBL_FTP_Download_files/analytics.js.download"></script><script>var prefetch = [];</script>
<script type="text/javascript" src="./ENSEMBL_FTP_Download_files/js(1)" async=""></script><script async="" src="./ENSEMBL_FTP_Download_files/hotjar-3280824.js.download"></script><style type="text/css">.fancybox-margin{margin-right:17px;}</style></head>
<body data-pace="2" class="static" id="ensembl-webpage" style="" cz-shortcut-listen="true">

  <div id="min_width_container">
    <div id="min_width_holder">
    
    <!-- Announcement Banner -->    
        <div class="info" id="announcement-banner"><h2>EMBL-EBI User Survey 2024</h2><p>Do data resources managed by EMBL-EBI and our collaborators make a difference to your work?</p><p>Please take 10 minutes to fill in our annual user survey, and help us make the case for why sustaining open data resources is critical for life sciences research.</p><p>Survey link: <a href="https://www.surveymonkey.com/r/HJKYKTT?channel=[webpage]">https://www.surveymonkey.com/r/HJKYKTT?channel=[webpage]</a></p></div>
    <!-- /Announcement Banner -->

      <div id="masthead" class="js_panel">
        <input type="hidden" class="panel_type" value="Masthead">
        <div class="logo_holder"><a href="https://plants.ensembl.org/"><img src="./ENSEMBL_FTP_Download_files/e.png" alt="Ensembl Genomes Home" title="Ensembl Genomes Home" class="print_hide" style="width:43px;height:40px"></a><a href="https://plants.ensembl.org/"><img src="./ENSEMBL_FTP_Download_files/ensemblPlants.png" alt="Ensembl Plants Home" title="Ensembl Plants Home" class="print_hide" style="width:173px;height:40px"></a><img src="./ENSEMBL_FTP_Download_files/e-ensembl_print.gif" alt="Ensembl Plants Home" title="Ensembl Plants Home" class="screen_hide_inline" style="width:173px;height:40px">
    <span class="print_hide">
      <span id="site_menu_button">▼</span>
      <ul id="site_menu" style="display:none" class="tab_context_menu">
        <li><a href="http://www.ensemblgenomes.org/">Ensembl Genomes</a></li>
        <li><a href="http://bacteria.ensembl.org/">Ensembl Bacteria</a></li>
        <li><a href="http://protists.ensembl.org/">Ensembl Protists</a></li>
        <li><a href="http://fungi.ensembl.org/">Ensembl Fungi</a></li>
        <li><a href="http://plants.ensembl.org/">Ensembl Plants</a></li>
        <li><a href="http://metazoa.ensembl.org/">Ensembl Metazoa</a></li>
        <li><a href="http://www.ensembl.org/">Ensembl (vertebrates)</a></li>
      </ul>
    </span>
  </div>
        <div class="mh print_hide">
          <div class="account_holder"><div class="_account_holder"><a class="constant account-link _accounts_no_user modal_link" href="https://plants.ensembl.org/Multi/Account/Login" title="Login/Register">Login/Register</a></div></div>
          <div class="tools_holder"><ul class="tools" style="width: auto;"><li><a class="constant" href="https://plants.ensembl.org/hmmer">HMMER</a></li><li><a class="constant" href="https://plants.ensembl.org/Multi/Tools/Blast">BLAST</a></li><li><a class="constant" href="https://plants.ensembl.org/biomart/martview">BioMart</a></li><li><a class="constant" href="https://plants.ensembl.org/tools.html">Tools</a></li><li><a class="constant" href="https://plants.ensembl.org/info/data/ftp/index.html">Downloads</a></li><li><a class="constant" href="https://plants.ensembl.org/info/">Help &amp; Docs</a></li><li class="last"><a class="constant" target="_blank" href="http://www.ensembl.info/">Blog</a></li></ul><div class="more" style="display: none;"><a href="https://plants.ensembl.org/info/data/ftp/index.html#">More <span class="arrow">▼</span></a><ul class="more_tools"></ul></div></div>
          <div class="search_holder print_hide">
    <div id="searchPanel" class="js_panel">
      <input type="hidden" class="panel_type" value="SearchBox">
      <form action="https://plants.ensembl.org/Multi/Psychic">
        <div class="search print_hide">
          <div class="sites button">
            <img class="search_image no-sprite badge-16" src="./ENSEMBL_FTP_Download_files/ensemblunit.gif" alt="">
            <img src="./ENSEMBL_FTP_Download_files/down.gif" style="width:7px" alt="">
            <input type="hidden" name="site" value="ensemblunit">
          </div>
          <div>
            <label class="hidden" for="se_q">Search terms</label>
            <input class="query inactive" id="se_q" type="text" name="q" value="Search Ensembl Plants…" data-role="none" onkeydown="if (event.keyCode == 13) { $(this).closest(&#39;form&#39;).submit(); return false; }">
          </div>
          <div class="button"><img src="./ENSEMBL_FTP_Download_files/search.png" alt="Search »" onclick="$(this).closest(&#39;form&#39;).submit()"></div>
        </div>
        <div class="site_menu hidden">
          <div class="ensemblunit"><img class="no-sprite" src="./ENSEMBL_FTP_Download_files/ensemblunit.gif" alt="Search Ensembl Plants">Search Ensembl Plants<input type="hidden" value="Search Ensembl Plants…"></div>
<div class="ensembl_genomes"><img class="no-sprite" src="./ENSEMBL_FTP_Download_files/ensembl_genomes.gif" alt="Search Ensembl Genomes">Search Ensembl Genomes<input type="hidden" value="Search Ensembl Genomes…"></div>
<div class="ensembl_all"><img class="no-sprite" src="./ENSEMBL_FTP_Download_files/ensembl.gif" alt="Search all species">Search all species<input type="hidden" value="Search all species…"></div>
<div class="ebi"><img class="no-sprite" src="./ENSEMBL_FTP_Download_files/ebi.gif" alt="Search EBI">Search EBI<input type="hidden" value="Search EBI…"></div>

        </div>
      </form>
    </div>
  </div>
        </div>
  
        
        <div class="tabs_holder print_hide"><ul class="tabs"><li class=" short_tab" style="display:none"><a href="https://plants.ensembl.org/info/website/" title="Using this website">Using</a></li><li class=" active short_tab" style="display:none"><a href="https://plants.ensembl.org/info/data/" title="Data access">Data</a></li><li class=" short_tab" style="display:none"><a href="https://plants.ensembl.org/info/docs/" title="API &amp; software">API</a></li><li class=" short_tab" style="display:none"><a href="https://plants.ensembl.org/info/about/" title="About us">About</a></li><li class=" long_tab" style="display: block;"><a href="https://plants.ensembl.org/info/website/">Using this website</a></li><li class=" active long_tab" style="display: block;"><a href="https://plants.ensembl.org/info/data/">Data access</a></li><li class=" long_tab" style="display: block;"><a href="https://plants.ensembl.org/info/docs/">API &amp; software</a></li><li class=" long_tab" style="display: block;"><a href="https://plants.ensembl.org/info/about/">About us</a></li></ul></div>
        
      </div>

      <div id="main_holder">
      
      <div id="page_nav_wrapper">
        <div id="page_nav" class="nav print_hide js_panel floating">
          
    <input type="hidden" class="panel_type" value="LocalContext">
    <div class="header">In this section</div>
    <ul class="local_context">
      <li class="last"><img src="./ENSEMBL_FTP_Download_files/leaf.gif" class="toggle rsync.html" alt=""><a href="https://plants.ensembl.org/info/data/ftp/rsync.html"><b>Downloading with rsync</b></a></li>
    </ul>
    
    
    
  
          
          
          <p class="invisible">.</p>
        </div>
      </div>
    

      <div id="main">
          <ul class="breadcrumbs print_hide"><li><a class="home" href="https://plants.ensembl.org/">Home</a></li><li><a href="https://plants.ensembl.org/info/">Help &amp; Documentation</a></li><li><a href="https://plants.ensembl.org/info/data/" title="Accessing Ensembl Data">Accessing Ensembl Data</a></li><li class="last"><a href="https://plants.ensembl.org/info/data/ftp/" title="FTP Download">FTP Download</a></li></ul>
          
          <div id="content"><div id="static">

<h1>FTP Download</h1>

<p>You can download via a browser from our <a href="https://ftp.ensemblgenomes.ebi.ac.uk/pub/plants" rel="external">FTP site</a>,
use a script, or even use <a href="https://plants.ensembl.org/info/data/ftp/rsync.html">rsync</a> from the command line.</p>

<h2>Globus</h2>

<p>For rapid bulk download of files, the Ensembl FTP site is available as an end point in the <a href="https://www.globus.org/" rel="external">Globus Online system</a>. In order to access the data you need to sign up for an account with Globus, install the Globus Connect Personal software and setup a personal endpoint to download the data.

The Ensembl data is hosted at the EMBL-EBI end point called "EMBL-EBI Public Data". Data from the Ensembl FTP site can then be found under the <code>"/ensemblorg/pub"</code>. You can also <a href="https://app.globus.org/file-manager?origin_id=47772002-3e5b-4fd3-b97c-18cee38d6df2&amp;origin_path=%2Fensemblorg%2Fpub%2F" rel="external">click here to open the target directory</a>.</p>


<h2>API Code</h2>

<p>If you do not have access to git, you can obtain our latest API code as a gzipped tarball:</p>

<p><a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/ensembl-api.tar.gz" rel="external">Download complete API for this release</a></p>

<p>Note: the API version needs to be the same as the databases you are accessing, so please
use git to obtain a previous version if querying older databases.</p>
    
<h2>Database dumps</h2>
<p>
Entire databases can be downloaded from our FTP site in a
variety of formats. Please be aware that some of these files
can run to many gigabytes of data.
</p>
<p><strong>Looking for <a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/current/mysql/" rel="external">MySQL dumps</a> to install databases locally?</strong> See our
<a href="https://www.ensembl.org/info/docs/webcode/mirror/install/ensembl-data.html" rel="external">web installation instructions</a>
for full details.</p>
<p>
Each directory on <a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub" rel="external">https://ftp.ebi.ac.uk/ensemblgenomes</a> contains a
<a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/current_README" rel="external">README</a> file, explaining the directory structure.
</p>
  <h3>Multi-species data</h3>
    <table class="ss autocenter" style="width: 100%" cellpadding="0" cellspacing="0">
      <thead><tr class="ss_header"><th class="sort_html" title="Database">Database</th><th class="sort_none" title="MySQL">MySQL</th><th class="sort_none" title="TSV">TSV</th><th class="sort_none" title="EMF">EMF</th><th class="sort_none" title="MAF">MAF</th><th class="sort_none" title="XML">XML</th></tr></thead>
      <tbody><tr class="bg1"><td><strong>Pan_compara Multi-species</strong></td><td><a rel="external" title="" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/pan_ensembl/release-59/mysql/ensembl_compara_pan_homology_59_112/">MySQL</a></td><td><a rel="external" title="" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/pan_ensembl/release-59/tsv/ensembl-compara/homologies">TSV</a></td><td><a rel="external" title="" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/pan_ensembl/release-59/emf/ensembl-compara/homologies">EMF</a></td><td></td><td><a rel="external" title="" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/pan_ensembl/release-59/xml/ensembl-compara/homologies">XML</a></td></tr><tr class="bg2"><td><strong>Plants Multi-species</strong></td><td><a rel="external" title="" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/plants/release-59/mysql/ensembl_compara_plants_59_112">MySQL</a></td><td><a rel="external" title="" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/plants/release-59/tsv/ensembl-compara/homologies" target="_blank">TSV</a></td><td><a rel="external" title="" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/plants/release-59/emf/ensembl-compara/homologies">EMF</a></td><td><a rel="external" title="" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/plants/release-59/maf/">MAF</a></td><td><a rel="external" title="" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/plants/release-59/xml/ensembl-compara/homologies">XML</a></td></tr><tr class="bg1"><td><strong>Ensembl Mart</strong></td><td><a rel="external" title="" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/plants/release-59/mysql/plants_mart_59">MySQL</a></td><td></td><td></td><td></td><td></td></tr></tbody>
    </table>
    
  <h3>Single species data</h3>
    <div class="js_panel" id="ftp-table">
      <input type="hidden" class="panel_type" value="Content">
      <p>Popular species are listed first. You can customise this list via our <a href="https://plants.ensembl.org/">home page</a>.</p>
      <div>
    <div class="dataTables_wrapper"><div class="dataTables_top"><div class="dataTables_length"><label>Show <select data-role="none" size="1"><option value="10" selected="selected">10</option><option value="25">25</option><option value="50">50</option><option value="100">100</option><option value="-1">All</option></select> entries</label></div><div class="col_toggle"><div class="toggle">Show/hide columns<ul class="floating_popup"><li><input data-role="none" type="checkbox" checked=""><span><img src="./ENSEMBL_FTP_Download_files/star.png"></span></li><li><input data-role="none" type="checkbox" checked=""><span>Species</span></li><li><input data-role="none" type="checkbox" checked=""><span>DNA (FASTA)</span></li><li><input data-role="none" type="checkbox" checked=""><span>cDNA (FASTA)</span></li><li><input data-role="none" type="checkbox" checked=""><span>CDS (FASTA)</span></li><li><input data-role="none" type="checkbox" checked=""><span>ncRNA (FASTA)</span></li><li><input data-role="none" type="checkbox" checked=""><span>Protein sequence (FASTA)</span></li><li><input data-role="none" type="checkbox" checked=""><span>Annotated sequence (EMBL)</span></li><li><input data-role="none" type="checkbox" checked=""><span>Annotated sequence (GenBank)</span></li><li><input data-role="none" type="checkbox" checked=""><span>Gene sets</span></li><li><input data-role="none" type="checkbox" checked=""><span>Other annotations</span></li><li><input data-role="none" type="checkbox" checked=""><span>Whole databases</span></li><li><input data-role="none" type="checkbox" checked=""><span>Variation (GVF)</span></li><li><input data-role="none" type="checkbox" checked=""><span>Variation (VCF)</span></li><li><input data-role="none" type="checkbox" checked=""><span>Variation (VEP)</span></li></ul></div></div><div class="dataTables_filter"><label><input data-role="none" type="text"><div class="overlay">Filter</div></label></div><div class="invisible"></div></div><table class="data_table autocenter ss" style="width: 100%" cellpadding="0" cellspacing="0">
      <thead><tr class="ss_header"><th class="sort_html sorting" title="Favourite" style="text-align:left;width:5%" rowspan="1" colspan="1"><img src="./ENSEMBL_FTP_Download_files/star.png"></th><th class="sort_html sorting" title="Species" style="width:10%;text-align:left" rowspan="1" colspan="1">Species</th><th class="sort_none sorting_disabled" title="DNA (FASTA)" style="text-align:center;width:10%" rowspan="1" colspan="1">DNA (FASTA)</th><th class="sort_none sorting_disabled" title="cDNA (FASTA)" style="text-align:center;width:10%" rowspan="1" colspan="1">cDNA (FASTA)</th><th class="sort_none sorting_disabled" title="CDS (FASTA)" style="text-align:center;width:10%" rowspan="1" colspan="1">CDS (FASTA)</th><th class="sort_none sorting_disabled" title="ncRNA (FASTA)" style="width:10%;text-align:center" rowspan="1" colspan="1">ncRNA (FASTA)</th><th class="sort_none sorting_disabled" title="Protein sequence (FASTA)" style="text-align:center;width:10%" rowspan="1" colspan="1">Protein sequence (FASTA)</th><th class="sort_none sorting_disabled" title="Annotated sequence (EMBL)" style="text-align:center;width:10%" rowspan="1" colspan="1">Annotated sequence (EMBL)</th><th class="sort_none sorting_disabled" title="Annotated sequence (GenBank)" style="text-align:center;width:10%" rowspan="1" colspan="1">Annotated sequence (GenBank)</th><th class="sort_none sorting_disabled" title="Gene sets" style="text-align:center;width:10%" rowspan="1" colspan="1">Gene sets</th><th class="sort_none sorting_disabled" title="Other annotations" style="width:10%;text-align:center" rowspan="1" colspan="1">Other annotations</th><th class="sort_none sorting_disabled" title="Whole databases" style="width:10%;text-align:center" rowspan="1" colspan="1">Whole databases</th><th class="sort_html sorting" title="Variation (GVF)" style="width:10%;text-align:center" rowspan="1" colspan="1">Variation (GVF)</th><th class="sort_html sorting" title="Variation (VCF)" style="width:10%;text-align:center" rowspan="1" colspan="1">Variation (VCF)</th><th class="sort_html sorting" title="Variation (VEP)" style="width:10%;text-align:center" rowspan="1" colspan="1">Variation (VEP)</th></tr></thead>
      
    <tbody><tr class="bg1"><td style="text-align:left;width:5%">Y</td><td style="width:10%;text-align:left"><b><a href="https://plants.ensembl.org/Arabidopsis_thaliana/">Arabidopsis thaliana</a></b></td><td style="width:10%;text-align:center"><a rel="external" title="Masked and unmasked genome sequences associated with the assembly (contigs, chromosomes etc.)" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/arabidopsis_thaliana/dna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="cDNA sequences for both Ensembl and &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/arabidopsis_thaliana/cdna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Coding sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/arabidopsis_thaliana/cds/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Non-coding RNA gene predictions" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/arabidopsis_thaliana/ncrna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Protein sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/arabidopsis_thaliana/pep/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Ensembl database dumps in EMBL nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/embl/arabidopsis_thaliana/">EMBL</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in GenBank nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/genbank/arabidopsis_thaliana/">GenBank</a></td><td style="text-align:center;width:10%"><a rel="external" title="Gene sets for each species. These files include annotations of both coding and non-coding genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gtf/arabidopsis_thaliana">GTF</a> <a rel="external" title="GFF3 provides access to all annotated transcripts which make up an Ensembl gene set" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gff3/arabidopsis_thaliana">GFF3</a></td><td style="width:10%;text-align:center"><a rel="external" title="External references in TSV format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/tsv/arabidopsis_thaliana">TSV</a> <a rel="external" title="External references and other annotation data in RDF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/json/arabidopsis_thaliana">JSON</a></td><td style="text-align:center;width:10%"><a rel="external" title="All Ensembl MySQL databases are available in text format as are the SQL table definition files" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/mysql/">MySQL</a></td><td style="width:10%;text-align:center"><a rel="external" title="Variation data in GVF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/gvf/arabidopsis_thaliana/">GVF</a></td><td style="width:10%;text-align:center"><a rel="external" title="Variation data in VCF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/vcf/arabidopsis_thaliana/">VCF</a></td><td style="text-align:center;width:10%"><a rel="external" title="Cache files for use with the VEP script" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/indexed_vep_cache/">VEP</a></td></tr><tr class="bg2"><td style="width:5%;text-align:left">Y</td><td style="text-align:left;width:10%"><b><a href="https://plants.ensembl.org/Oryza_sativa/">Oryza sativa Japonica Group</a></b></td><td style="text-align:center;width:10%"><a rel="external" title="Masked and unmasked genome sequences associated with the assembly (contigs, chromosomes etc.)" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/oryza_sativa/dna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="cDNA sequences for both Ensembl and &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/oryza_sativa/cdna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Coding sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/oryza_sativa/cds/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Non-coding RNA gene predictions" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/oryza_sativa/ncrna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Protein sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/oryza_sativa/pep/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in EMBL nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/embl/oryza_sativa/">EMBL</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in GenBank nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/genbank/oryza_sativa/">GenBank</a></td><td style="text-align:center;width:10%"><a rel="external" title="Gene sets for each species. These files include annotations of both coding and non-coding genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gtf/oryza_sativa">GTF</a> <a rel="external" title="GFF3 provides access to all annotated transcripts which make up an Ensembl gene set" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gff3/oryza_sativa">GFF3</a></td><td style="width:10%;text-align:center"><a rel="external" title="External references in TSV format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/tsv/oryza_sativa">TSV</a> <a rel="external" title="External references and other annotation data in RDF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/json/oryza_sativa">JSON</a></td><td style="text-align:center;width:10%"><a rel="external" title="All Ensembl MySQL databases are available in text format as are the SQL table definition files" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/mysql/">MySQL</a></td><td style="width:10%;text-align:center"><a rel="external" title="Variation data in GVF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/gvf/oryza_sativa/">GVF</a></td><td style="width:10%;text-align:center"><a rel="external" title="Variation data in VCF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/vcf/oryza_sativa/">VCF</a></td><td style="text-align:center;width:10%"><a rel="external" title="Cache files for use with the VEP script" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/indexed_vep_cache/">VEP</a></td></tr><tr class="bg1"><td style="text-align:left;width:5%">Y</td><td style="text-align:left;width:10%"><b><a href="https://plants.ensembl.org/Triticum_aestivum/">Triticum aestivum</a></b></td><td style="width:10%;text-align:center"><a rel="external" title="Masked and unmasked genome sequences associated with the assembly (contigs, chromosomes etc.)" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/triticum_aestivum/dna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="cDNA sequences for both Ensembl and &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/triticum_aestivum/cdna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Coding sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/triticum_aestivum/cds/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Non-coding RNA gene predictions" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/triticum_aestivum/ncrna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Protein sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/triticum_aestivum/pep/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Ensembl database dumps in EMBL nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/embl/triticum_aestivum/">EMBL</a></td><td style="text-align:center;width:10%"><a rel="external" title="Ensembl database dumps in GenBank nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/genbank/triticum_aestivum/">GenBank</a></td><td style="width:10%;text-align:center"><a rel="external" title="Gene sets for each species. These files include annotations of both coding and non-coding genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gtf/triticum_aestivum">GTF</a> <a rel="external" title="GFF3 provides access to all annotated transcripts which make up an Ensembl gene set" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gff3/triticum_aestivum">GFF3</a></td><td style="text-align:center;width:10%"><a rel="external" title="External references in TSV format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/tsv/triticum_aestivum">TSV</a> <a rel="external" title="External references and other annotation data in RDF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/json/triticum_aestivum">JSON</a></td><td style="width:10%;text-align:center"><a rel="external" title="All Ensembl MySQL databases are available in text format as are the SQL table definition files" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/mysql/">MySQL</a></td><td style="text-align:center;width:10%"><a rel="external" title="Variation data in GVF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/gvf/triticum_aestivum/">GVF</a></td><td style="width:10%;text-align:center"><a rel="external" title="Variation data in VCF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/vcf/triticum_aestivum/">VCF</a></td><td style="width:10%;text-align:center"><a rel="external" title="Cache files for use with the VEP script" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/indexed_vep_cache/">VEP</a></td></tr><tr class="bg2"><td style="text-align:left;width:5%">Y</td><td style="text-align:left;width:10%"><b><a href="https://plants.ensembl.org/Hordeum_vulgare/">Hordeum vulgare</a></b></td><td style="text-align:center;width:10%"><a rel="external" title="Masked and unmasked genome sequences associated with the assembly (contigs, chromosomes etc.)" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/hordeum_vulgare/dna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="cDNA sequences for both Ensembl and &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/hordeum_vulgare/cdna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Coding sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/hordeum_vulgare/cds/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Non-coding RNA gene predictions" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/hordeum_vulgare/ncrna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Protein sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/hordeum_vulgare/pep/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Ensembl database dumps in EMBL nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/embl/hordeum_vulgare/">EMBL</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in GenBank nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/genbank/hordeum_vulgare/">GenBank</a></td><td style="text-align:center;width:10%"><a rel="external" title="Gene sets for each species. These files include annotations of both coding and non-coding genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gtf/hordeum_vulgare">GTF</a> <a rel="external" title="GFF3 provides access to all annotated transcripts which make up an Ensembl gene set" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gff3/hordeum_vulgare">GFF3</a></td><td style="text-align:center;width:10%"><a rel="external" title="External references in TSV format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/tsv/hordeum_vulgare">TSV</a> <a rel="external" title="External references and other annotation data in RDF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/json/hordeum_vulgare">JSON</a></td><td style="text-align:center;width:10%"><a rel="external" title="All Ensembl MySQL databases are available in text format as are the SQL table definition files" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/mysql/">MySQL</a></td><td style="text-align:center;width:10%"><a rel="external" title="Variation data in GVF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/gvf/hordeum_vulgare/">GVF</a></td><td style="text-align:center;width:10%"><a rel="external" title="Variation data in VCF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/vcf/hordeum_vulgare/">VCF</a></td><td style="text-align:center;width:10%"><a rel="external" title="Cache files for use with the VEP script" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/indexed_vep_cache/">VEP</a></td></tr><tr class="bg1"><td style="text-align:left;width:5%">Y</td><td style="text-align:left;width:10%"><b><a href="https://plants.ensembl.org/Zea_mays/">Zea mays</a></b></td><td style="width:10%;text-align:center"><a rel="external" title="Masked and unmasked genome sequences associated with the assembly (contigs, chromosomes etc.)" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/zea_mays/dna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="cDNA sequences for both Ensembl and &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/zea_mays/cdna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Coding sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/zea_mays/cds/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Non-coding RNA gene predictions" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/zea_mays/ncrna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Protein sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/zea_mays/pep/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in EMBL nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/embl/zea_mays/">EMBL</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in GenBank nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/genbank/zea_mays/">GenBank</a></td><td style="text-align:center;width:10%"><a rel="external" title="Gene sets for each species. These files include annotations of both coding and non-coding genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gtf/zea_mays">GTF</a> <a rel="external" title="GFF3 provides access to all annotated transcripts which make up an Ensembl gene set" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gff3/zea_mays">GFF3</a></td><td style="width:10%;text-align:center"><a rel="external" title="External references in TSV format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/tsv/zea_mays">TSV</a> <a rel="external" title="External references and other annotation data in RDF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/json/zea_mays">JSON</a></td><td style="text-align:center;width:10%"><a rel="external" title="All Ensembl MySQL databases are available in text format as are the SQL table definition files" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/mysql/">MySQL</a></td><td style="text-align:center;width:10%"><a rel="external" title="Variation data in GVF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/gvf/zea_mays/">GVF</a></td><td style="text-align:center;width:10%"><a rel="external" title="Variation data in VCF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/vcf/zea_mays/">VCF</a></td><td style="text-align:center;width:10%"><a rel="external" title="Cache files for use with the VEP script" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/indexed_vep_cache/">VEP</a></td></tr><tr class="bg2"><td style="text-align:left;width:5%">Y</td><td style="text-align:left;width:10%"><b><a href="https://plants.ensembl.org/Physcomitrium_patens/">Physcomitrium patens</a></b></td><td style="text-align:center;width:10%"><a rel="external" title="Masked and unmasked genome sequences associated with the assembly (contigs, chromosomes etc.)" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/physcomitrium_patens/dna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="cDNA sequences for both Ensembl and &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/physcomitrium_patens/cdna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Coding sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/physcomitrium_patens/cds/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Non-coding RNA gene predictions" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/physcomitrium_patens/ncrna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Protein sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/physcomitrium_patens/pep/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in EMBL nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/embl/physcomitrium_patens/">EMBL</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in GenBank nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/genbank/physcomitrium_patens/">GenBank</a></td><td style="width:10%;text-align:center"><a rel="external" title="Gene sets for each species. These files include annotations of both coding and non-coding genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gtf/physcomitrium_patens">GTF</a> <a rel="external" title="GFF3 provides access to all annotated transcripts which make up an Ensembl gene set" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gff3/physcomitrium_patens">GFF3</a></td><td style="width:10%;text-align:center"><a rel="external" title="External references in TSV format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/tsv/physcomitrium_patens">TSV</a> <a rel="external" title="External references and other annotation data in RDF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/json/physcomitrium_patens">JSON</a></td><td style="width:10%;text-align:center"><a rel="external" title="All Ensembl MySQL databases are available in text format as are the SQL table definition files" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/mysql/">MySQL</a></td><td style="width:10%;text-align:center">-</td><td style="width:10%;text-align:center">-</td><td style="width:10%;text-align:center"><a rel="external" title="Cache files for use with the VEP script" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/indexed_vep_cache/">VEP</a></td></tr><tr class="bg1"><td style="width:5%;text-align:left"></td><td style="width:10%;text-align:left"><b><a href="https://plants.ensembl.org/Actinidia_chinensis/">Actinidia chinensis</a></b></td><td style="width:10%;text-align:center"><a rel="external" title="Masked and unmasked genome sequences associated with the assembly (contigs, chromosomes etc.)" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/actinidia_chinensis/dna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="cDNA sequences for both Ensembl and &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/actinidia_chinensis/cdna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Coding sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/actinidia_chinensis/cds/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Non-coding RNA gene predictions" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/actinidia_chinensis/ncrna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Protein sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/actinidia_chinensis/pep/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Ensembl database dumps in EMBL nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/embl/actinidia_chinensis/">EMBL</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in GenBank nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/genbank/actinidia_chinensis/">GenBank</a></td><td style="text-align:center;width:10%"><a rel="external" title="Gene sets for each species. These files include annotations of both coding and non-coding genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gtf/actinidia_chinensis">GTF</a> <a rel="external" title="GFF3 provides access to all annotated transcripts which make up an Ensembl gene set" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gff3/actinidia_chinensis">GFF3</a></td><td style="text-align:center;width:10%"><a rel="external" title="External references in TSV format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/tsv/actinidia_chinensis">TSV</a> <a rel="external" title="External references and other annotation data in RDF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/json/actinidia_chinensis">JSON</a></td><td style="width:10%;text-align:center"><a rel="external" title="All Ensembl MySQL databases are available in text format as are the SQL table definition files" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/mysql/">MySQL</a></td><td style="text-align:center;width:10%">-</td><td style="width:10%;text-align:center">-</td><td style="width:10%;text-align:center"><a rel="external" title="Cache files for use with the VEP script" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/indexed_vep_cache/">VEP</a></td></tr><tr class="bg2"><td style="text-align:left;width:5%"></td><td style="width:10%;text-align:left"><b><a href="https://plants.ensembl.org/Aegilops_tauschii/">Aegilops tauschii</a></b></td><td style="text-align:center;width:10%"><a rel="external" title="Masked and unmasked genome sequences associated with the assembly (contigs, chromosomes etc.)" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/aegilops_tauschii/dna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="cDNA sequences for both Ensembl and &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/aegilops_tauschii/cdna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Coding sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/aegilops_tauschii/cds/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Non-coding RNA gene predictions" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/aegilops_tauschii/ncrna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Protein sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/aegilops_tauschii/pep/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in EMBL nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/embl/aegilops_tauschii/">EMBL</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in GenBank nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/genbank/aegilops_tauschii/">GenBank</a></td><td style="width:10%;text-align:center"><a rel="external" title="Gene sets for each species. These files include annotations of both coding and non-coding genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gtf/aegilops_tauschii">GTF</a> <a rel="external" title="GFF3 provides access to all annotated transcripts which make up an Ensembl gene set" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gff3/aegilops_tauschii">GFF3</a></td><td style="text-align:center;width:10%"><a rel="external" title="External references in TSV format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/tsv/aegilops_tauschii">TSV</a> <a rel="external" title="External references and other annotation data in RDF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/json/aegilops_tauschii">JSON</a></td><td style="text-align:center;width:10%"><a rel="external" title="All Ensembl MySQL databases are available in text format as are the SQL table definition files" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/mysql/">MySQL</a></td><td style="text-align:center;width:10%">-</td><td style="width:10%;text-align:center">-</td><td style="text-align:center;width:10%"><a rel="external" title="Cache files for use with the VEP script" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/indexed_vep_cache/">VEP</a></td></tr><tr class="bg1"><td style="text-align:left;width:5%"></td><td style="text-align:left;width:10%"><b><a href="https://plants.ensembl.org/Aegilops_umbellulata/">Aegilops umbellulata</a></b></td><td style="text-align:center;width:10%"><a rel="external" title="Masked and unmasked genome sequences associated with the assembly (contigs, chromosomes etc.)" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/aegilops_umbellulata/dna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="cDNA sequences for both Ensembl and &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/aegilops_umbellulata/cdna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Coding sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/aegilops_umbellulata/cds/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Non-coding RNA gene predictions" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/aegilops_umbellulata/ncrna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Protein sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/aegilops_umbellulata/pep/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Ensembl database dumps in EMBL nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/embl/aegilops_umbellulata/">EMBL</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in GenBank nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/genbank/aegilops_umbellulata/">GenBank</a></td><td style="width:10%;text-align:center"><a rel="external" title="Gene sets for each species. These files include annotations of both coding and non-coding genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gtf/aegilops_umbellulata">GTF</a> <a rel="external" title="GFF3 provides access to all annotated transcripts which make up an Ensembl gene set" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gff3/aegilops_umbellulata">GFF3</a></td><td style="text-align:center;width:10%"><a rel="external" title="External references in TSV format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/tsv/aegilops_umbellulata">TSV</a> <a rel="external" title="External references and other annotation data in RDF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/json/aegilops_umbellulata">JSON</a></td><td style="width:10%;text-align:center"><a rel="external" title="All Ensembl MySQL databases are available in text format as are the SQL table definition files" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/mysql/">MySQL</a></td><td style="text-align:center;width:10%">-</td><td style="text-align:center;width:10%">-</td><td style="text-align:center;width:10%"><a rel="external" title="Cache files for use with the VEP script" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/indexed_vep_cache/">VEP</a></td></tr><tr class="bg2"><td style="text-align:left;width:5%"></td><td style="text-align:left;width:10%"><b><a href="https://plants.ensembl.org/Amborella_trichopoda/">Amborella trichopoda</a></b></td><td style="width:10%;text-align:center"><a rel="external" title="Masked and unmasked genome sequences associated with the assembly (contigs, chromosomes etc.)" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/amborella_trichopoda/dna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="cDNA sequences for both Ensembl and &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/amborella_trichopoda/cdna/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Coding sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/amborella_trichopoda/cds/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Non-coding RNA gene predictions" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/amborella_trichopoda/ncrna/">FASTA</a></td><td style="text-align:center;width:10%"><a rel="external" title="Protein sequences for Ensembl or &quot;ab initio&quot; predicted genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/fasta/amborella_trichopoda/pep/">FASTA</a></td><td style="width:10%;text-align:center"><a rel="external" title="Ensembl database dumps in EMBL nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/embl/amborella_trichopoda/">EMBL</a></td><td style="text-align:center;width:10%"><a rel="external" title="Ensembl database dumps in GenBank nucleotide sequence database format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/genbank/amborella_trichopoda/">GenBank</a></td><td style="width:10%;text-align:center"><a rel="external" title="Gene sets for each species. These files include annotations of both coding and non-coding genes" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gtf/amborella_trichopoda">GTF</a> <a rel="external" title="GFF3 provides access to all annotated transcripts which make up an Ensembl gene set" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/gff3/amborella_trichopoda">GFF3</a></td><td style="text-align:center;width:10%"><a rel="external" title="External references in TSV format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/tsv/amborella_trichopoda">TSV</a> <a rel="external" title="External references and other annotation data in RDF format" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/json/amborella_trichopoda">JSON</a></td><td style="text-align:center;width:10%"><a rel="external" title="All Ensembl MySQL databases are available in text format as are the SQL table definition files" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/mysql/">MySQL</a></td><td style="text-align:center;width:10%">-</td><td style="width:10%;text-align:center">-</td><td style="width:10%;text-align:center"><a rel="external" title="Cache files for use with the VEP script" href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/release-59/plants/variation/indexed_vep_cache/">VEP</a></td></tr></tbody></table><div class="dataTables_bottom"><div class="dataTables_info">Showing 1 to 10 of 153 entries</div><div class="dataTables_paginate paging_full_numbers"><span class="first paginate_button paginate_button_disabled">&lt;&lt;</span><span class="previous paginate_button paginate_button_disabled">&lt;</span><span><span class="paginate_active">1</span><span class="paginate_button">2</span><span class="paginate_button">3</span><span class="paginate_button">4</span><span class="paginate_button">5</span></span><span class="next paginate_button">&gt;</span><span class="last paginate_button">&gt;&gt;</span></div><div class="invisible"></div></div></div>
    <div class="data_table_config"><input type="hidden" name="code" value="FTPtable::153"><input type="hidden" name="iDisplayLength" value="10"><input type="hidden" name="expopts" value="[]"></div>
  </div>
    </div>
    
<h3>Metadata</h3>

<p>Data files containing metadata for Ensembl Genomes from release 15 onwards can be found 
in the root directory or appropriate division directory of each release e.g.
<a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/current/plants/" rel="external">https://ftp.ebi.ac.uk/ensemblgenomes/pub/current/plants/</a>.</p>

<p>The following files are provided:</p>

<ul>
  <li><a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/current/species.txt" rel="external">species.txt</a> (or e.g. <a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/current/plants/species_EnsemblPlants.txt" rel="external">species_EnsemblPlants.txt</a>) - simple tab-separated file containing basic information about each genome</li>
  <li><a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/current/species_metadata.json" rel="external">species_metadata.json</a> (or e.g. <a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/current/plants/species_metadata_EnsemblPlants.json" rel="external">species_metadata_EnsemblPlants.json</a>) - full metadata about each genome in JSON format, including comparative analyses, sequence region names etc.</li>
  <li><a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/current/species_metadata.xml" rel="external">species_metadata.xml</a> (or e.g. <a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/current/plants/species_metadata_EnsemblPlants.xml" rel="external">species_metadata_EnsemblPlants.xml</a>) - full metadata about each genome in XML format, including comparative analyses, sequence region names etc.</li>
  <li><a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/current/uniprot_report.txt" rel="external">uniprot_report.txt</a> (or e.g. <a href="https://ftp.ebi.ac.uk/ensemblgenomes/pub/current/plants/uniprot_report_EnsemblPlants.txt" rel="external">uniprot_report_EnsemblPlants.txt</a>) - specialised tab-separated file containing information about mapping of genome to UniProtKB</li>
</ul>
  
    
    <p>
To facilitate storage and download all databases are
<a href="http://directory.fsf.org/project/gzip/" rel="external">GNU
Zip</a> (gzip, *.gz) compressed.
</p>

<h2>About the data</h2>

<p>
The following types of data dumps are available on the FTP site.
</p>

<dl class="twocol striped">
<dt class="bg2">FASTA</dt>
<dd class="bg2">FASTA sequence databases of Ensembl gene, transcript and protein
model predictions. Since the
<a href="http://www.bioperl.org/wiki/FASTA_sequence_format" rel="external">FASTA format</a> does not permit sequence annotation,
these database files are mainly intended for use with local sequence
similarity search algorithms. Each directory has a README file with a
detailed description of the header line format and the file naming
conventions.
<dl>
  <dt>DNA</dt>
  <dd><a href="http://www.repeatmasker.org/" rel="external">Masked</a>
  and unmasked genome sequences associated with the assembly (contigs,
  chromosomes etc.).</dd>
  <dd>The header line in an FASTA dump files containing DNA sequence
  consists of the following attributes :
  coord_system:version:name:start:end:strand
  This coordinate-system string is used in the Ensembl API to retrieve
  slices with the SliceAdaptor.</dd>

  <dt>CDS</dt>
  <dd>Coding sequences for Ensembl or <i>ab
  initio</i> <a href="https://www.ensembl.org/info/genome/genebuild/" rel="external">predicted
  genes</a>.</dd>

  <dt>cDNA</dt>
  <dd>cDNA sequences for Ensembl or <i>ab
  initio</i> <a href="https://www.ensembl.org/info/genome/genebuild/" rel="external">predicted
  genes</a>.</dd>

  <dt>Peptides</dt>
  <dd>Protein sequences for Ensembl or <i>ab
  initio</i> <a href="https://www.ensembl.org/info/genome/genebuild/" rel="external">predicted
  genes</a>.</dd>


  <dt>RNA</dt>
  <dd>Non-coding RNA gene predictions.</dd>

</dl>

</dd>

<dt class="bg1">Annotated sequence</dt>
<dd class="bg1">Flat files allow more extensive sequence annotation by means of
feature tables and contain thus the genome sequence as annotated by
the automated Ensembl
<a href="https://www.ensembl.org/info/genome/genebuild/" rel="external">genome
annotation pipeline</a>. Each nucleotide sequence record in a flat
file represents a 1Mb slice of the genome sequence. Flat files are
broken into chunks of 1000 sequence records for easier downloading.
  <dl>

  <dt>EMBL</dt>
  <dd>Ensembl database dumps in <a href="http://www.ebi.ac.uk/ena/about/sequence_format" rel="external">EMBL</a> nucleotide
  sequence <a href="https://ftp.ebi.ac.uk/pub/databases/embl/doc/usrman.txt" rel="external">database format</a></dd>

  <dt>GenBank</dt>
  <dd>Ensembl database dumps
  in <a href="http://www.ncbi.nlm.nih.gov/genbank/" rel="external">GenBank</a> nucleotide sequence
  <a href="http://www.ncbi.nlm.nih.gov/Sitemap/samplerecord.html" rel="external">database format</a></dd>

  </dl>

</dd>

<dt class="bg2">MySQL</dt>
<dd class="bg2">All Ensembl <a href="http://www.mysql.com/" rel="external">MySQL</a> databases are available in text format as are
the SQL table definition files. These can be imported into any SQL
database for a local
<a href="https://www.ensembl.org/info/docs/webcode/mirror/install/" rel="external">installation</a> of a mirror
site. Generally, the FTP directory tree contains one directory per
database. For more information about these databases and their
Application Programming Interfaces (or APIs) see the
<a href="https://www.ensembl.org/info/docs/api/" rel="external">API</a> section.</dd>

<dt class="bg1">GTF</dt>
<dd class="bg1">Gene sets for each species. These files include annotations of
both coding and non-coding genes. This file format is
described <a href="http://www.gencodegenes.org/pages/data_format.html" rel="external">here</a>.
</dd>

<dt class="bg1">GFF3</dt>
<dd class="bg1">GFF3 provides access to all annotated transcripts which make
up an Ensembl gene set. This file format is
described <a href="http://www.sequenceontology.org/gff3.shtml" rel="external">here</a>.
</dd>

  
<dt class="bg2">EMF flatfile dumps (comparative data)</dt>
<dd class="bg2">
<p>
Alignments of resequencing data are available for several species as
Ensembl Multi Format (EMF) flatfile dumps. The accompanying README
file describes the file format.
</p>

<p>
Also, the same format is used to dump whole-genome multiple alignments
as well as gene-based multiple alignments and phylogentic trees used
to infer Ensembl orthologues and paralogues. These files are available
in the ensembl_compara database which will be found in
the <a href="https://plants.ensembl.org/info/data/ftp/[[SPECIESDEFS::ENSEMBL_FTP_URL]]/current_mysql/">mysql
directory</a>.
</p>
</dd>

<dt class="bg2">MAF (comparative data)</dt>
<dd class="bg2">
<p>
MAF files are provided for all pairwise alignments containing human
(GRCh38), and all multiple alignments.
The MAF file format is described <a href="http://genome.ucsc.edu/FAQ/FAQformat.html#format5" rel="external">here</a>.
</p>
</dd>
      
<dt class="bg1">GVF (variation data)</dt>
<dd class="bg1">GVF (Genome Variation Format) is a simple tab-delimited format derived
from GFF3 for variation positions across the genome.
There are GVF files for different types of variation data (e.g.
somatic variants, structural variants etc). For more information see
the "README" files in the GVF directory.</dd>

<dt class="bg2">VCF (variation data)</dt>
<dd class="bg2">VCF (Variant Call Format) is a text file format containing meta-information lines, a header
line, and then data lines each containing information about a position in the genome. This file format can also contain genotype information on samples for each position.
More details about the format and its specifications are available <a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41" rel="external">here</a>.</dd>


<dt class="bg1">VEP (variation data)</dt>
<dd class="bg1">Compressed text files (called "cache files") used by the <a href="https://plants.ensembl.org/VEP">Variant Effect Predictor</a> tool. More information about these files is available <a href="https://www.ensembl.org/info/docs/tools/vep/script/vep_cache.html" rel="external">here</a>.</dd>

<dt class="bg2">BED format files (comparative data)</dt>
<dd class="bg2">
<p>
Constrained elements calculated using GERP are available in BED
format. For more information see the accompanying README file.
</p>

<p>
BED format is a simple line-based format. The first 3 mandatory columns
are:
</p>

<ul>
<li>chromosome name (may start with 'chr' for compliance with UCSC)</li>
<li>start position. This is a 0-based position</li>
<li>end position.</li>
</ul>

<p>
<a href="https://plants.ensembl.org/info/website/upload/bed.html">More information on the BED file format</a>...
</p>
</dd>
      
<dt class="bg1">Tarball</dt>
<dd class="bg1">
<p>
The entire Ensembl API is gzipped and concatenated into a single TAR file. This is updated daily.</p>
</dd>
      </dl>
  


</div></div>
      </div>
  
        <div id="footer">
          <div class="column-wrapper">
  <div class="column-two left">
		  <p>
      Ensembl Plants release 59 - May 2024
		  © <span class="print_hide"><a href="http://www.ebi.ac.uk/" style="white-space:nowrap">EMBL-EBI</a></span>
      <span class="screen_hide_inline">EMBL-EBI</span>
      </p>
  </div>
            <p class="invisible">.</p>
          </div>
          <div class="column-wrapper"><hr><div id="fat-footer">
              <div class="column-four left">
                <h3>About Us</h3>
                <p><a href="https://plants.ensembl.org/info/about/">About us</a></p>
                <p><a href="https://plants.ensembl.org/info/about/contact/">Contact us</a></p>
                <p><a href="https://plants.ensembl.org/info/about/publications.html">Citing Ensembl Genomes</a></p>
                <p><a href="https://www.ebi.ac.uk/data-protection/ensembl/privacy-notice">Privacy policy</a></p>
                <p><a href="https://plants.ensembl.org/info/about/legal/">Disclaimer</a></p>
              </div>
  
              <div class="column-four left">
                <h3>Get help</h3>
                <p><a href="https://plants.ensembl.org/info/website/">Using this website</a></p>
                <p><a href="https://plants.ensembl.org/info/">Documentation</a></p>
                <p><a href="https://plants.ensembl.org/info/website/upload/">Adding custom tracks</a></p>
                <p><a href="https://plants.ensembl.org/info/data/">Downloading data</a></p>
              </div>
  
              <div class="column-four left">
                <h3>Our sister sites</h3>
                <p><a href="http://www.ensembl.org/">Ensembl</a></p><p><a href="http://bacteria.ensembl.org/">Ensembl Bacteria</a></p><p><a href="http://fungi.ensembl.org/">Ensembl Fungi</a></p><p><a href="http://protists.ensembl.org/">Ensembl Protists</a></p><p><a href="http://metazoa.ensembl.org/">Ensembl Metazoa</a></p>
              </div>
  
              <div class="column-four left">
                <h3>Follow us</h3>
                <p><a class="media-icon" href="http://www.ensembl.info/">
                  <img alt="[RSS logo]" title="Ensembl blog" src="./ENSEMBL_FTP_Download_files/rss_icon_16.png"></a>
                  <a href="http://www.ensembl.info/">Blog</a></p>
                <p><a class="media-icon" href="https://twitter.com/ensemblgenomes">
                  <img alt="[twitter logo]" title="Follow us on Twitter!" src="./ENSEMBL_FTP_Download_files/twitter.png"></a>
                    <a href="https://twitter.com/ensemblgenomes">Twitter</a></p>
              </div>
  </div>
            <p class="invisible">.</p>
          </div>
        </div>
  
      </div>
    </div>
  </div>
  <form id="core_params" action="https://plants.ensembl.org/info/data/ftp/index.html#" style="display:none">
    <fieldset><input type="hidden" name="db" value="core"></fieldset>
  </form>
  <input type="hidden" id="species_path" name="species_path" value="">
  <input type="hidden" id="gdpr_version" name="gdpr_version" value="2.0.0">
  <input type="hidden" id="gdpr_policy_url" name="gdpr_policy_url" value="https://www.ebi.ac.uk/data-protection/ensembl/privacy-notice">
  <input type="hidden" id="gdpr_terms_url" name="gdpr_terms_url" value="https://www.ebi.ac.uk/about/terms-of-use">
  <input type="hidden" id="gdpr_cookie_name" name="gdpr_cookie_name" value="ENSEMBL_PRIVACY_POLICY">
  <input type="hidden" id="species_common_name" name="species_common_name" value="Oryza sativa Japonica Group">
  <input type="hidden" id="ensembl_image_root" name="ensembl_image_root" value="/i/">
  <input type="hidden" id="max_region_length" name="max_region_length" value="100010">
  <input type="hidden" id="ensembl_species_image" name="ensembl_species_image" value="/i/species/Oryza_sativa.png">
    
  <div id="modal_bg"></div>
  <div id="modal_panel" class="js_panel ui-draggable ui-resizable" style="height: 1926px; width: 957px; top: 241px; left: 53px;">
    <input type="hidden" class="panel_type" value="ModalContainer">
    <div class="modal_title">
      <ul class="tabs">
        <li><a class="modal_user_data" href="https://plants.ensembl.org/info/data/ftp/UserData/ManageData?time=1716373776">Personal Data</a></li>
      </ul>
      <div class="modal_caption"></div>
      <div class="modal_close"></div>
    </div>
    <div id="modal_user_data" class="modal_content js_panel fixed_width active" style="display:none"></div>
    <div id="modal_default" class="modal_content js_panel fixed_width" style="display:none"></div>
    <div class="modal_overlay"><img class="overlay_close" title="Cancel" alt="close" src="./ENSEMBL_FTP_Download_files/close.png"><div class="overlay_content"></div></div>
    <div class="modal_overlay_bg"></div>
  <div class="ui-resizable-handle ui-resizable-e" style="z-index: 90;"></div><div class="ui-resizable-handle ui-resizable-s" style="z-index: 90;"></div><div class="ui-resizable-handle ui-resizable-se ui-icon ui-icon-gripsmall-diagonal-se" style="z-index: 90;"></div></div>
  
    
    <script type="text/javascript" src="./ENSEMBL_FTP_Download_files/61b5041d0ed2cc2c618aa65a3cd2f3f0.js.download"></script>
<script type="text/javascript" src="./ENSEMBL_FTP_Download_files/c283ba204795b53f3bf5f2517ec23786.js.download"></script>

  <input type="hidden" id="hidden_species_list" name="hidden_species_list" value="Oryza_sativa_lima|Avena_sativa_OT3098|Arabis_alpina|Citrullus_lanatus|Oryza_sativa_mh63|Selaginella_moellendorffii|Dioscorea_rotundata|Fraxinus_excelsior|Oryza_glumipatula|Aegilops_tauschii|Setaria_italica|Corylus_avellana|Triticum_aestivum_norin61|Trifolium_pratense|Triticum_aestivum|Papaver_somniferum|Cucumis_sativus|Vigna_angularis|Ananas_comosus|Amborella_trichopoda|Triticum_aestivum_arinalrfor|Oryza_sativa_khaoyaiguang|Beta_vulgaris|Oryza_sativa_arc|Triticum_aestivum_weebil|Oryza_nivara|Lolium_perenne|Sesamum_indicum|Eucalyptus_grandis|Oryza_sativa_ir64|Brassica_napus|Physcomitrium_patens|Cucumis_melo|Ostreococcus_lucimarinus|Digitaria_exilis|Sorghum_bicolor|Arabidopsis_thaliana|Chlamydomonas_reinhardtii|Triticum_aestivum_robigus|Cannabis_sativa_female|Chara_braunii|Oryza_sativa_natelboro|Manihot_esculenta|Panicum_hallii_fil2|Olea_europaea|Nymphaea_colorata|Theobroma_cacao_criollo|Oryza_indica|Corchorus_capsularis|Ficus_carica|Oryza_sativa_gobolsailbalam|Leersia_perrieri|Nicotiana_attenuata|Oryza_sativa_chaomeo|Quercus_lobata|Cajanus_cajan|Prunus_persica|Gossypium_raimondii|Secale_cereale|Oryza_sativa_larhamugad|Musa_acuminata|Saccharum_spontaneum|Cyanidioschyzon_merolae|Panicum_hallii|Triticum_urartu|Marchantia_polymorpha|Prunus_avium|Daucus_carota|Oryza_sativa_ketannangka|Brassica_oleracea|Oryza_brachyantha|Kalanchoe_fedtschenkoi|Lupinus_angustifolius|Prunus_dulcis|Vigna_unguiculata|Triticum_aestivum_julius|Triticum_aestivum_jagger|Chondrus_crispus|Vicia_faba|Triticum_spelta|Echinochloa_crusgalli|Brachypodium_distachyon|Oryza_sativa_liuxu|Avena_sativa_Sang|Oryza_sativa_n22|Olea_europaea_sylvestris|Pistacia_vera|Solanum_tuberosum|Corymbia_citriodora|Eutrema_salsugineum|Brassica_rapa_ro18|Populus_trichocarpa|Phaseolus_vulgaris|Triticum_aestivum_paragon|Coffea_canephora|Glycine_soja|Oryza_longistaminata|Eragrostis_tef|Juglans_regia|Solanum_tuberosum_RH8903916|Vigna_radiata|Triticum_aestivum_mattis|Triticum_aestivum_renan|Oryza_sativa|Hordeum_vulgare_TRITEX|Brassica_juncea|Asparagus_officinalis|Triticum_aestivum_cadenza|Brassica_rapa|Oryza_barthii|Triticum_dicoccoides|Citrus_clementina|Actinidia_chinensis|Glycine_max|Triticum_aestivum_kariega|Oryza_sativa_zs97|Eragrostis_curvula|Oryza_sativa_pr106|Vitis_vinifera|Oryza_sativa_azucena|Hordeum_vulgare_goldenpromise|Galdieria_sulphuraria|Arabidopsis_lyrata|Zea_mays|Cynara_cardunculus|Oryza_glaberrima|Triticum_aestivum_stanley|Triticum_turgidum|Helianthus_annuus|Lactuca_sativa|Triticum_aestivum_mace|Arabidopsis_halleri|Oryza_rufipogon|Medicago_truncatula|Setaria_viridis|Triticum_aestivum_claire|Oryza_meridionalis|Ipomoea_triloba|Chenopodium_quinoa|Quercus_suber|Malus_domestica_golden|Solanum_lycopersicum|Oryza_punctata|Pisum_sativum|Aegilops_umbellulata|Triticum_aestivum_landmark|Triticum_aestivum_lancer|Triticum_aestivum_refseqv2|Theobroma_cacao|Camelina_sativa|Hordeum_vulgare|Rosa_chinensis|Capsicum_annuum">


<div class="cookie-message" style="display: block;"><p class="msg">This website requires cookies, and the limited processing of your personal data in order to function. By using the site you are agreeing to this as outlined in our <a target="_blank" href="https://www.ebi.ac.uk/data-protection/ensembl/privacy-notice">Privacy Policy</a> and <a target="_blank" href="https://www.ebi.ac.uk/about/terms-of-use"> Terms of Use </a></p><div class="agree-button"><a id="gdpr-agree" class="button no-underline"> I Agree </a></div></div></body></html>